source/sass/patterns/molecules/figure.scss

```diff
@@ -0,0 +1,19 @@
+@import "../../mixins/spacing";
+@import "../../mixins/typography";
+
+.figure {
+  @include block-spacing($end: $baselinegrid-space-medium);
```

Comment:
One of the few places where we do need to set `$start` to be the same. (Not sure there's another approach?)

Reply:
Won't the spacing before the figure come from the bottom of the preceding block?

Reply:
Ideally, but there's no way in CSS about knowing what the following sibling is...

Reply:
Fair. Agree `$baselinegrid-space-medium` needs to go on the start and the end in this case.